cmd/oceanbench/set.go

```diff
@@ -189,6 +201,35 @@ func writeDevices(w http.ResponseWriter, r *http.Request, msg string, args ...in
 		return
 	}
 
+	// Provide uptime and device status information.
```

Comment:
Why has this chunk of code moved?

Reply:
since the sandbox site page does not need all the device variables, moving this code here prevents calls to the database for unused variables, and instead only gets the ones that matter first.